pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -73,6 +73,8 @@
         "controller.validation.frequencyInSeconds";
     private static final String OFFLINE_SEGMENT_INTERVAL_CHECKER_FREQUENCY_IN_SECONDS =
         "controller.offline.segment.interval.checker.frequencyInSeconds";
+    private static final String SEGMENT_VALIDATION_INITIAL_DELAY_IN_SECONDS =
```

Comment:
should this be called REALTIME_SEGMENT_VALIDATION_INITIAL_DELAY_IN_SECONDS to match other variable conventions(like REALTIME_SEGMENT_VALIDATION_FREQUENCY_IN_SECONDS)

Reply:
All segment-level validation are done with a single call to get a random back-off. Therefore it makes sense to have a single variable. RealtimeSegemntValidationManager calls getPeriodicTasksInitialDelay(). Do we really want to introduce(even) more configs just for test sake?